modules/hitlnext/src/views/full/app/Reducer.ts

```diff
@@ -53,7 +53,10 @@ const Reducer = (state: IState, action: ActionType): IState => {
       return produce(state, draft => {
         draft.handoffs = {
           ...draft.handoffs,
-          [action.payload.id]: _.merge(draft.handoffs[action.payload.id], action.payload.payload)
+          [action.payload.id]: {
+            ..._.merge(draft.handoffs[action.payload.id], action.payload.payload),
+            tags: action.payload.payload.tags
```

Comment:
@EFF @allardy There was an issue here when removing a tag. Since it this "merges" the old tags with the new ones, the resulting array would contain the removed item. This seems to fix the issue, but is kind of hackish imho. Any suggestion on how I could make this cleaner?